runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -95,6 +95,7 @@ template <typename LoaderDomain, typename SpecificLoader> class BaseLoader
   template <typename Param> void loadPool2D(Param &param, const Pool2DOptions *options);
 
   // Operations
+  void LoadAddV2(const Operator *op, ir::Graph &subg);
```

Comment:
Where is its implementation?
it shows only forward declare.

Reply:
In the loadCustom function, AddV2 calls loadAdd(). Therefore, we will remove LoadAddV2.